src/css-skew.js

```diff
@@ -14,6 +14,20 @@
 
 (function(internal, scope) {
 
+  function computeMatrix(cssSkew) {
+    // CSSSkew represented by the 2D matrix:
+    //   1     tan(ax)  0
+    // tan(ay)    1     0
+
+    var tanAx = CSSSkew._tanDegrees(cssSkew.ax);
```

Comment:
That's actually a little more complicated than it looks due to the way that skew is tested. I'll do it in a separate PR.